### ISSUE TITLE
Simple audio tutorial

### ISSUE THREAD

**Issue by dvisockas** [pytorch/tutorials#925]
Hello,

Seems like pytorch-audio is an under-served part of pytorch in its tutorials page.

I'd like to contribute by writing an initial tutorial starting with a simple audio classification (VCTK or UrbanSound for starters).

What are the requirements for that? Are there any guidelines? Maybe there are people working on torch-audio I could ping if something is not clear?

**Comment by vincentqb** [comment]
Thanks for reaching out! Feel free to add "cc @vincentqb" if you want to ping my attention directly here :)

Yes, that would be a very nice addition to the tutorials. I wrote the initial torchaudio tutorial in #572 by creating a jupyter notebook (e.g. [notebook](https://github.com/vincentqb/audio-tutorial/blob/master/Tutorial.ipynb)) and then converting it using [this script](https://gist.github.com/chsasank/7218ca16f8d022e02a9c0deb94a310fe). I then did some adjusting on the final rst file directly.

I'd say a good tutorial should be fairly short to run, say less than ~2 minutes. It's not a hard stop, but, unless there is something exciting to look at while code is running, it may not keep people's attention as much as a shorter tutorial to run.

I'm myself looking into adding a tutorial inspired by [the wav2letter pipeline](https://github.com/pytorch/audio/tree/master/examples/pipeline_wav2letter) in torchaudio.
